src/game_initialization/lobby_info.hpp

```diff
@@ -179,7 +179,8 @@ enum notify_mode {
 	NOTIFY_WHISPER_OTHER_WINDOW,
 	NOTIFY_LOBBY_JOIN,
 	NOTIFY_LOBBY_QUIT,
-	NOTIFY_COUNT
+	NOTIFY_COUNT,
```

Comment:
If an enum's final entry is `..._COUNT` then it's likely to be a way to get the number of entries in the enum, so any new entries should be added before it.

Nothing uses `LOBBY_COUNT`, so let's remove it.